editor/store/index.js

```diff
@@ -12,7 +12,10 @@ import { withRehydratation, loadAndPersist } from './persist';
 import enhanceWithBrowserSize from './mobile';
 import applyMiddlewares from './middlewares';
 import { BREAK_MEDIUM } from './constants';
-import { getEditedPostTitle } from './selectors';
+import {
+	getEditedPostContent,
+	getEditedPostTitle,
+} from './selectors';
```

Comment:
I think we should find a place to document the exposed selectors for plugin consumption.

Reply:
Good catch. I somehow missed that. It would be awesome to automate it using JSDoc :D